src/engine/Camera.ts

```diff
@@ -170,6 +178,35 @@ export class RadiusAroundActorStrategy implements CameraStrategy<Actor> {
   };
 }
 
+/**
+ *  Prevent a camera from going beyond the given camera dimensions.
+ */
+export class LimitCameraBoundsStrategy implements CameraStrategy<BoundingBox> {
```

Comment:
@DaVince this is great, exactly what I was thinking for implementation

Reply:
Cool, I'll start writing some tests then!